en/components/query-builder.md

```diff
@@ -149,25 +149,34 @@ The `expressionTree` is a two-way bindable property which means a corresponding
 ## Expressions Dragging
 
 With the [*Drag & Drop*](drag-drop.md) functionality, users can easily reposition condition chips to adjust their query logic dynamically.
-#### How It Works
-1. **Click and Hold:** Click and hold on chip with `right mouse` button.
-2. **Drag to Reposition:** Drag away with mouse, a drag ghost will appear. Move it to a desired position within the query, a drop ghost will appear on the designated new position.
-3. **Drop to Apply Changes:** Release the mouse button to drop the chip in its new location.
-4. **Observe Updates:** The query logic updates dynamically to reflect the new order.
-
+Chips can be dragged along the whole query tree, including in groups and subgroups. 
 ##### Notes
 - Dragging a chip does not modify its condition, only its position.
 - In the main query, changes take effect immediately after dropping a chip. In inner queries, user has to commit the inner query after repositioning the chip. 
+- Chip can also be dragged in a newly created group to achieve condition grouping.
```

Comment:
Chip can also be dragged in a newly created group to achieve condition grouping. -> [For example]
Group/ungroup expressions is handled through the Drag and Drop functionality. 
In order to group already existing conditions, first you need to add a new group through the 'add' group button. Then via drag and drop, the required expressions could be moved to the group:
[gif here]
In order to ungroup, you could drag all conditions outside their current group and once the last condition is moved out the group would be deleted.